scripts/ccf/member/info.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+REPO_ROOT="$(realpath "$(dirname "$(realpath "${BASH_SOURCE[0]}")")/../../..")"
+
+ccf-member-info() {
+
+    source $REPO_ROOT/scripts/ccf/member/id.sh
+
+    MEMBER_NAME=${1:-$MEMBER_NAME}
+    if [ -z "$MEMBER_NAME" ]; then
+        read -p "Enter member name: " MEMBER_NAME
+    fi
+    export MEMBER_NAME
+
+    # First check if the member is active
+    export MEMBER_ID=`ccf-member-id $MEMBER_NAME`
+    if_active_member=`curl -s $KMS_URL/gov/service/members?api-version=2024-07-01 \
+        --cacert $KMS_SERVICE_CERT_PATH \
+        | jq -e ".value[] | select(.memberId==\"$MEMBER_ID\")"`
+
+    if [ -n "$if_active_member" ]; then
+        echo $if_active_member
+        return
+    fi
+    export MEMBER_CERT=$(cat $WORKSPACE/${MEMBER_NAME}_cert.pem)
```

Comment:
Is it possible to do `export` later to make the interface clearer?
I'm thinking about having lines at the bottom of the function like:
```
export MEMBER_NAME
export MEMBER_CERT
...
```

This line can be just `MEMBER_CERT=$(cat $WORKSPACE/${MEMBER_NAME}_cert.pem)` without export.

Reply:
I think the tricky thing with that scheme is there are multiple exit points so it wouldn't allow us to move the exporting much lower down, not sure the benefits are worth it?

Reply:
Ah, that's true. I agree with you.

Reply:
Maybe it might be worth thinking about moving these script to something like python which can handle states with class easily (i.e. just by following conventional ways) , but I guess not now